python_ta/reporters/node_printers.py

```diff
@@ -14,7 +14,11 @@ def render_generic(msg, source_lines=None):
     if hasattr(msg, 'node') and msg.node is not None:
         node = msg.node
         start_line, start_col = node.fromlineno, node.col_offset
-        end_line, end_col = node.end_lineno, node.end_col_offset
+
+        if isinstance(node, astroid.FunctionDef) or isinstance(node, astroid.ClassDef):
```

Comment:
@lorena-b this can be simplified---look up the docs for `isinstance`, as it can take a tuple as a second parameter.